toolchain/semantics/testdata/operators/binary_op.carbon

```diff
@@ -6,27 +6,25 @@
 // RUN: %{carbon-run-semantics}
 // CHECK:STDOUT: cross_reference_irs.size == 2,
 // CHECK:STDOUT: cross_references = {
-// CHECK:STDOUT:   node_xref0 = "xref(ir0, block0, node0)";
-// CHECK:STDOUT:   node_xref1 = "xref(ir0, block0, node1)";
+// CHECK:STDOUT:   node_xref0 = xref(ir0, block0, node0);
+// CHECK:STDOUT:   node_xref1 = xref(ir0, block0, node1);
+// CHECK:STDOUT:   node_xref2 = xref(ir0, block0, node2);
+// CHECK:STDOUT:   node_xref3 = xref(ir0, block0, node3);
 // CHECK:STDOUT: },
 // CHECK:STDOUT: identifiers = {
 // CHECK:STDOUT:   ident0 = "Main";
 // CHECK:STDOUT: },
-// CHECK:STDOUT: integer_literals = {
-// CHECK:STDOUT:   int0 = 12;
-// CHECK:STDOUT:   int1 = 34;
-// CHECK:STDOUT: },
 // CHECK:STDOUT: node_blocks = {
 // CHECK:STDOUT:   block0 = {
 // CHECK:STDOUT:     node0 = FunctionDeclaration();
 // CHECK:STDOUT:     node1 = BindName(ident0, node0);
 // CHECK:STDOUT:     node2 = FunctionDefinition(node0, block1);
 // CHECK:STDOUT:   },
 // CHECK:STDOUT:   block1 = {
-// CHECK:STDOUT:     node0 = IntegerLiteral(int0): node_xref1;
-// CHECK:STDOUT:     node1 = IntegerLiteral(int1): node_xref1;
```

Comment:
I'm assuming removing some of the int literal stuff was intended here and just will return later?

Maybe worth a mention in the commit description? Clearly not a big deal as this is still early iteration.

Reply:
It's the 2nd and 3rd paragraphs of the description

Reply:
I just missed it, sorry!

But I'm happy either way here. Let's have a longer discussion around the best way to manage the connection between SIR and the lower layers (both the parse tree and token buffer) -- I can at least share some of the things that we ended up bumping into in Clang with a bunch of different use cases and maybe that helps shed light on what the right design is here.